backend/src/server/LogError.ts

```diff
@@ -0,0 +1,9 @@
+import { backendLogger as logger } from './logger'
+
+export default class LogError extends Error {
```

Comment:
Did you think about modifying the Error class itself? So we avoid those terrible imports.

Reply:
I thought about it - but its advised never to do that, even tho possible. This is the proper way to do it. Further more its not required to expose anything to the client.

We can log any data without exposing it to the client.